concent_api/core/views.py

```diff
@@ -358,20 +365,28 @@ def handle_receive_force_get_task_result_upload_for_requestor(decoded_message: m
     assert decoded_message.TYPE in message.registered_message_types
 
     current_time = int(datetime.datetime.now().timestamp())
-    decoded_message.timestamp                                      = current_time
-    decoded_message.file_transfer_token.timestamp                  = current_time
-    decoded_message.file_transfer_token.token_expiration_deadline  = current_time + settings.TOKEN_EXPIRATION_DEADLINE
-    decoded_message.file_transfer_token.operation                  = 'download'
+    file_transfer_token = message.concents.FileTransferToken(
+        token_expiration_deadline    = current_time + settings.TOKEN_EXPIRATION_DEADLINE,
+        storage_cluster_address      = decoded_message.file_transfer_token.storage_cluster_address,
+        authorized_client_public_key = decoded_message.file_transfer_token.authorized_client_public_key,
+        operation                    = 'download',
+    )
+    file_transfer_token.files = decoded_message.file_transfer_token.files
+
+    force_get_task_result_upload = message.concents.ForceGetTaskResultUpload(
+        file_transfer_token   = file_transfer_token,
+    )
+    force_get_task_result_upload.force_get_task_result = decoded_message.force_get_task_result
```

Comment:
Wouldn't it be possible to add this to the constructor call too?

Reply:
:+1:

Reply:
What about this one? Did you manage to do it?

Reply:
No, it raises `E:389, 8: Instance of 'ForceGetTaskResultUpload' has no 'force_get_task_result' member (no-member)`. I don't remember why I have marked it as ok.